.eslintrc.json

```diff
@@ -8,7 +8,7 @@
   },
   "settings": {
     "react": {
-      "version": "16.4"
+      "version": "17.2"
```

Comment:
Unrelated, but as internally we only support 17 make this match. It is prompted by eslint-config-skyscanner, though React isn't actually used in this repo at all.

Reply:
I tried removing this locally to see what happens:

```
 alakbarzeynalzade@M-LC2R24049H  ~/dev/babel-plugin-transform-i18n   add_gha ±  npm run lint          

> @skyscanner/babel-plugin-skyscanner-i18n@1.1.1 lint
> eslint *.js

Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.
```

Reply:
It doesn't _break_ anything, but it is an annoying warning, so I can understand why _some_ version was added as a default - may as well keep that up to date even though its currently non-functional :D